app/services/Agents.scala

```diff
@@ -104,12 +104,21 @@ class Agents @Inject() (amiableConfigProvider: AmiableConfigProvider, lifecycle:
   }
 
   def refreshOldInstanceCountInfo(instancesWithAmis: List[(Instance, Option[AMI])]): Unit = {
-    val now = DateTime.now
-    val oldInstanceCountsByAccount = PrismLogic.oldInstances(instancesWithAmis)
-      .groupBy(_.meta.origin.accountName.getOrElse("unknown-account"))
-      .map{ case (accountName, list) => OldInstanceAccountHistory(now, accountName, list.length)}
-      .toList
-    oldInstanceCountByAccountAgent.send(oldInstanceCountsByAccount)
+    for {
+      accounts <- Prism.getAccounts
```

Comment:
sometimes when there's only one thing in a for, yield I just do a `.flatMap (so Prism.getAccounts.flatMap(accounts=>` but that might noot be possible on an `Attempt`